packages/app/src/utils/useEnsureRinkebyConnect.tsx

```diff
@@ -5,27 +5,27 @@ import { Box, Button, Card, CardFooter, CardHeader, Heading, Layer } from "gromm
 type InferArguments<T> = T extends (...t: [...infer Arg]) => any ? Arg : never;
 type InferReturn<T> = Promise<T extends (...t: [...infer Res]) => infer Res ? Res : never>;
 
-export const useForceRinkebyFunction = <TFunc extends (...args: any[]) => any>(
+export const useEnsureRinkebyConnect = <TFunc extends (...args: any[]) => any>(
   func: TFunc
 ): {
   rinkebyForcedFunction: (...args: InferArguments<TFunc>) => InferReturn<TFunc>;
   renderError: () => ReactElement;
 } => {
-  const { chainId } = useEthers();
-  const [wrongNetwork, setWrongNetwork] = useState(false);
+  const { account, chainId } = useEthers();
+  const [displayWarning, setdisplayWarning] = useState(false);
   const onClose = useCallback((e) => {
     e.preventDefault();
-    setWrongNetwork(false);
+    setdisplayWarning(false);
   }, []);
 
   const renderError = () => {
-    if (wrongNetwork === false) return <></>;
+    if (!displayWarning) return <></>;
     return (
       <Layer style={{ overflow: "auto" }} animation="fadeIn" onEsc={onClose} onClickOutside={onClose}>
         <Card flex={false} pad="medium" width="large">
           <CardHeader justify="center" pad="none">
             <Heading level={2} alignSelf="center">
-              {"Please switch to rinkeby to use Tenderize"}
+              {account == null ? "Please connect your wallet" : "Please switch to rinkeby to use Tenderize"}
```

Comment:
```suggestion
              {!account ? "Please connect your wallet" : "Please switch to rinkeby to use Tenderize"}
```

Reply:
@kyriediculous added this because it kind of make sense here, but in general I consider it an anti-pattern, because it evaluates truthy falsy values - making  `!0` and `!""` true, instead of checking if they are `null` / `undefined`.